### DISCUSSION TITLE
bull v4.10.2 and bullmq v3.5.0 ?

### DISCUSSION THREAD

**Discussion by xeoshow** [OptimalBits/bull#2503] [Q&A]
Hello,

we are just deciding between bull and bullmq, for the latest verson of both, bull v4.10.2 and bullmq v3.5.0, from stability and performance and usability perspective, which one would you like to recommend? 

Thanks a lot.

Best regards, Jason

**Comment by manast** [comment]
At this point I would recommend BullMQ, it is production ready, and its where most of our efforts are spent today. Also, if you can, try to use typescript.
